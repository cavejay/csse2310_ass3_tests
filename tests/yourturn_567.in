newround 7
yourturn 5
yourturn 6
yourturn 7
gameover

