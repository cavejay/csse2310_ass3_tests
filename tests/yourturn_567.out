-7--
5A-
7--
